### PR TITLE
show error when creating the product

### LINKED ISSUE #966: Create Product page doesn't show error
Screen Shot 2021-10-14 at 11 46 37 AM

┆Issue is synchronized with this [Monday item](https://seed-company-squad.monday.com/boards/3451697530/pulses/3466803762) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by elisabeth-sorrell**
Closes https://github.com/SeedCompany/cord-field/issues/966

┆Issue is synchronized with this [Monday item](https://seed-company-squad.monday.com/boards/3451697530/pulses/3466497337) by [Unito](https://www.unito.io)
┆Link To Item: https://seed-company-squad.monday.com/boards/3451697530/pulses/3466497337

### CODE REVIEW

**Comment by elisabeth-sorrell** [comment]
Screen Shot 2022-05-19 at 6 54 57 PM

File: src/scenes/Products/Create/CreateProduct.tsx [hunk 1]
```diff
@@ -208,10 +209,19 @@ export const CreateProduct = () => {
         <EngagementBreadcrumb data={engagement} />
         <Typography variant="h4">Create Goal</Typography>
       </Breadcrumbs>
-      <Typography variant="h2">
-        {loading ? <Skeleton width="50%" variant="text" /> : 'Create Goal'}
-      </Typography>
+      <Error error={error}>
+        {{
+          NotFound: 'Could not find Engagement',
+          Default: 'Error loading Engagement',
```
Comment (CarsonF) [inline src/scenes/Products/Create/CreateProduct.tsx:215]: Can this reference goal creation somehow? Maybe
```suggestion
          NotFound: 'Cannot create goal for this unknown engagement',
          Default: 'Failed to load engagement info needed to create goal',
```

**Review by CarsonF — COMMENTED** [review]
> I personally don't really like keeping the breadcrumb, so I can take that out too if you agree.

Normally I'd say we should try to be consistent with layout. If it's not pulsing, I'd say it's fine.

**Comment by elisabeth-sorrell** [comment]
> > I personally don't really like keeping the breadcrumb, so I can take that out too if you agree.
> 
> Normally I'd say we should try to be consistent with layout. If it's not pulsing, I'd say it's fine.

Took a look to check and it is pulsing. What do we usually do for this kind of situation in field?

**Comment by sethmcknight** [comment]
This lived to the end of its lifetime and died

**Pull request closed without merging**